omnibus-base-goerli.toml

```diff
@@ -102,9 +114,14 @@ args = [
 ]
 depends = ["invoke.createScPool", "invoke.createEthSynth"]
 
-# TODO clean this block after deployment
-[invoke.unapprove_wrongfully_deployed_snx_pool]
+[invoke.configure_weth_collateral_for_spartan_council_pool]
 target = ["system.CoreProxy"]
-fromCall.func = "owner"
-func = "removeApprovedPool"
-args = ["2"]
+fromCall.func = "getPoolOwner"
+fromCall.args = ["<%= settings.sc_pool_id %>"]
+func = "setPoolCollateralConfiguration"
+args = [
+    "<%= settings.sc_pool_id %>",
+    "<%= settings.weth_address %>",
+    { collateralLimitD18 = "<%= MaxUint256 %>", issuanceRatioD18 = "<%= MaxUint256 %>" }
```

Comment:
Is `issuanceRatioD18` set to MaxUInt on purpose to have issuance turned off?

Reply:
Or maybe that means that is will allow what ever is configured for the collateral it self?

Reply:
> Is `issuanceRatioD18` set to MaxUInt on purpose to have issuance turned off?

yes